talk/concurrency/threadsasync.tex

```diff
@@ -86,7 +86,7 @@
       \begin{itemize}
       \item execution starts when \mintinline{cpp}{get()} is called on the returned future
       \end{itemize}
-    \item default is not specified!
+    \item default is not specified, but tries to use existing concurrency!
```

Comment:
What does that mean? :) What is "existing concurrency"? Will it reuse the existing thread, which was spawned before somewhere else? Will it use an existing OpenMP threadpool? (probably not).

Could you maybe phrase this a bit more specifically. At least I would not know what to explain here :)

Reply:
From [cppreference](https://en.cppreference.com/w/cpp/thread/async):
> If more than one flag is set, it is implementation-defined which policy is selected. For the default (both the [std::launch::async](https://en.cppreference.com/w/cpp/thread/launch) and [std::launch::deferred](https://en.cppreference.com/w/cpp/thread/launch) flags are set in policy), standard recommends (but doesn't require) utilizing available concurrency, and deferring any additional tasks

Reply:
My takeaway is that it will use existing threads if possible, and defer other tasks. If TBB is at the backend, it will most likely just schedule a task and let TBB handle the rest.